src/main/resources/scss/abstracts/_overrides.scss

```diff
@@ -22,9 +22,8 @@
 
 // Remove when code blocks follows user's theme
 // https://github.com/jenkinsci/design-library-plugin/issues/68
-pre {
+pre.jdl-component-code__code {
```

Comment:
This ensures that the important styles here are not applied to the codemirror styles (#103)